zebrad/src/components/sync/downloads.rs

```diff
@@ -113,7 +113,7 @@ where
     #[instrument(level = "debug", skip(self), fields(%hash))]
     pub async fn download_and_verify(&mut self, hash: block::Hash) -> Result<(), Report> {
         if self.cancel_handles.contains_key(&hash) {
-            return Err(eyre!("duplicate hash queued for download"));
+            return Err(eyre!("duplicate hash queued for download: {:?}", hash));
```

Comment:
I think it might be appropriate to add a feature for disabling issue reporting at the point of creation, something like this...

```suggestion
            return Err(eyre!("duplicate hash queued for download: {:?}", hash))
                .generate_issue_url(false);
```

That way we don't need to keep doing string comparisons for filtering

Reply:
I opened #1527 for this